docs/components/integration-framework/connectors/custom-built-connectors/connector-sdk.md

```diff
@@ -4,17 +4,681 @@ title: Connector SDK
 ---
 
+import Tabs from "@theme/Tabs";
+import TabItem from "@theme/TabItem";
+
 :::note
 The **connector SDK** is in developer preview and subject to breaking changes. Use at your own risk.
 :::
 
-The **connector SDK** allows you to develop your own **connectors**. With the help of the SDK you can perform common operations, such as:
+The **connector SDK** allows you to [develop custom connectors](#creating-a-custom-connector)
+using Java code. You can focus on the logic of the connector, conveniently test it locally, and
+reuse its runtime logic in multiple environments. The SDK achieves this by abstracting from
+Camunda Platform 8 internals that usually come with
+[job workers](../../../concepts/job-workers.md).
+
+The SDK provides APIs for common connector operations, such as:
+
+- Fetching and deserializing input data
+- Validating input data
+- Replacing secrets in input data
+
+Beyond that, the SDK allows for [convenient testing](#testing) of your connector behavior and
+[executing it in the environments](#runtime-environments) that suit your use cases best.
+
+## Creating a custom connector
+
+Using the connector SDK, you can create environment-agnostic, reusable connector runtime behavior.
+This section will outline how to set up a connector project, test it, and run it locally.
+
+### Setup
+
+When developing a new connector we recommend using our
+[custom connector template repository on GitHub](https://github.com/camunda/connector-template).
+This template is a [Maven](https://maven.apache.org/)-based Java project, and can be used in various
+ways as described below:
+
+- _Create your own GitHub repository_: Click the **Use this template** button and follow the steps.
+  You can manage code changes in your new repository afterward.
+- _Experiment locally_: Check out the source code to your local machine using [Git](https://git-scm.com/).
+  You won't be able to check in code changes to the repository due to restricted write access.
+- _Fetch the source_: Download the source code as a ZIP archive using **Code** > **Download ZIP**.
+  You can adjust and manage the code the way you like afterward, using your chosen source code
+  management tools.
+
+If you want to manualy set up your connector project, include the following dependency to use the SDK.
+Make sure to adhere to the project outline detailed in the next section.
+
+<Tabs groupId="dependency" defaultValue="maven" values={
+[
+{label: 'Maven dependency', value: 'maven' },
+{label: 'Gradle dependency', value: 'gradle' }
+]
+}>
+
+<TabItem value='maven'>
+
+```xml
+<dependency>
+  <groupId>io.camunda.connector</groupId>
+  <artifactId>connector-core</artifactId>
+  <version>0.1.0</version>
+</dependency>
+```
+
+</TabItem>
+
+<TabItem value='gradle'>
+
+```yml
+implementation 'io.camunda.connector:connector-core:0.1.0'
+```
+
+</TabItem>
+</Tabs>
+
+### Project outline
+
+There are multiple parts to a connector that enable it to be reused, as a
```

Comment:
Does "There are multiple parts to a connector that enable it to be reused, as a reusable building block..." mean two different things, or do you mean "There are multiple parts to a connector that enables it for reuse, for modeling, and for runtime behavior."

Reply:
I see this is confusing.

I propose to simplify this by splitting into two sentences:
```There are multiple parts to a connector: the runtime logic and a modeling configuration. These two combined make it a reuseable building block for the integration with external systems.
The following shows the setup of a connector in more detail:```